tests/test_http_auth.py

```diff
@@ -0,0 +1,170 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2022-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import base64
+import struct
+
+from edgedb import scram
+
+from edb import protocol
+from edb.testbase import http as tb
+from edb.testbase import server as tb_server
+
+
+class TestHttpAuth(tb.BaseHttpTest, tb_server.ConnectedTestCase):
+    @classmethod
+    def get_api_path(cls) -> str:
+        return "/"
+
+    def test_http_auth_scram(self):
+        args = self.get_connect_args()
+
+        with self.http_con() as con:
+            _, headers, status = self.http_con_request(
+                con, {}, path="auth/scram"
```

Comment:
Is there a point in duplicating the auth scheme in path when we already indicate it in headers?

Reply:
So just `GET /auth`? Or like @jaclarke suggested, maybe something like `GET /auth_token`? I'm okay with either. Please advise.

Reply:
Let's go with `GET /auth/token`